apps/api/.env.example

```diff
@@ -4,6 +4,9 @@ SERVER_PORT=
 # Node env
 NODE_ENV= # Use "development" for graphql playground to work
 
+# Upload folder location
+Upload_Path=./src/uploads/
+
 # Database configuration
 DB_TYPE=
 DB_HOST=
```

Comment:
@ramraut-osm 
Can you add this change in your PR instead. And fix the prettier lintings.
https://github.com/OsmosysSoftware/osm-transcript-summarizer/pull/17#discussion_r1597449894

Reply:
Done

Reply:
Env naming is still incorrect.

Reply:
Changed it to UPLOAD_FOLDER_PATH